contents/blog/vc-or-bootstrap.md

```diff
@@ -0,0 +1,101 @@
+
+The reality is that you can sell secondaries, but they aren't guaranteed. Most founders we know do this around the series B, or whenever the company is an obvious success. For employees, this normally starts a couple of rounds later. You can't sell anything like 100% of your stake each time - a few percent of what you own is the norm - VCs want you to remove some personal stress so you can focus on the business, but they still want you hungry for more. As the company gets bigger, this can go from removing any personal debt, to a life changing amount _whilst_ still running the company.
+
+Tim and I both think this path will give us the greatest happiness. It means removing financial stress from our lives, whilst being lucky enough to work on our mission of increasing the number of successful products in the world. It's a huge amount of fun for me to try to beat huge competitors by doing things "the right way" for developers.
+
+## Getting started
+
+If you want to get started with bootstrapping, you have two choices:
+
+* start working on your new business before you quit your job
+* spend your personal capital until you're profitable
+
+Neither of these sounds like a relaxing exercise!
+
+We started by bootstrapping. The reality at PostHog is that Tim and I committed 12 months of personal runway to getting off the ground. We simply paid ourselves nothing for the first 6 months. That created stress - we had to save up ahead of time, then had to spend as little as possible in our personal lives.
+
+Then we got into [YCombinator](https://ycombinator.com). The personal stress evaporated and we focused 100% on the business.
+
+What I didn't realize at the time is many teams will raise a pre-seed round, ahead of real traction or having a team - this reduces the barrier of needing a bunch of savings.
+
+## Some companies are a terrible fit for VC
+
+First, you need to want to build something very, very big. You will never be able to deliberately relax and run things as a lifestyle business once you're at $1M ARR or more, for example. Are you going to want to push further when you've no financial need to do so?
+
+Second, the style of product you want to build may _require_ more capital. With hindsight, we've built a really wide platform with lots of products inside it. The innovation is starting on an open source base, and integrating all the tools that customers need to build a more successful product. That's an awful lot of stuff to build. We also believed that our paid self hosted product would be 90%+ of our revenue, so we'd need to build a free and a paid product on top (although that later turned out not to be true...)
+
+## There are terrible reasons for raising money...
+
+Although there may be exceptions, here are standard ways to fail:
+
+* You want to hire people to find product market fit for you
+* You want to hire people to do the early sales for you
+* You want to raise money because of your ego
```

Comment:
```suggestion
* You're seeking a sky-high valuation
```

Take or leave this, but in infers the same thing.

Reply:
i'll leave it (but thank you for the other suggestions here!)